init.lua

```diff
@@ -252,7 +263,7 @@ minetest.register_privilege("rank", {
 
 -- Assign/update rank on join player
 minetest.register_on_joinplayer(function(player)
-	if ranks.get_rank(player) then
+	if storage:get_string(player:get_player_name()) then
```

Comment:
No change should be necessary here due to retaining the check in `ranks.get_rank` that ensures that the argument passed in is always a string or if not, is converted to one.